Sources/protoc-gen-swift/MessageFieldGenerator.swift

```diff
@@ -275,10 +275,12 @@ extension Google_Protobuf_FieldDescriptorProto {
 struct MessageFieldGenerator {
     private let fieldDescriptor: FieldDescriptor
     private let generatorOptions: GeneratorOptions
+    private let namer: SwiftProtobufNamer
 
     var descriptor: Google_Protobuf_FieldDescriptorProto { return fieldDescriptor.proto }
     let oneof: Google_Protobuf_OneofDescriptorProto?
     let jsonName: String?
+    let hasFieldPresence: Bool
```

Comment:
Performance is not a primary concern for code generation, so I'd prefer whatever is clearest.